response_operations_ui/templates/collection-exercise.html

```diff
@@ -231,7 +245,11 @@ <h2 class="venus u-pt-m">Sample</h2>
               </tr>
             </tbody>
           </table>
-        <p>Sample loaded: {{sample.ingestDateTime}}</p>
+        <p>Sample loaded: {{sample.ingestDateTime}}
+          {% if show_edit_period %}
```

Comment:
Can we replace all uses of show_edit_period and `if not ce.state == 'Live' and not ce.state == 'Ready for live'` checks with `if not locked` as they do not take 'EXECUTION_STARTED', 'VALIDATED', 'EXECUTED' states under consideration.